news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Kyoku: Original Song & Beatmapping Contest"
+date: 2024-05-10 18:00:00 +0000
+---
+
+Mappers and musicians, gather up! Express your creativity to the maximum with mapping and music, right here on osu!.
+
+![](/wiki/shared/news/2024-05-10-kyoku-original-song-and-beatmapping-contest/banner.jpg)
+
+Brought to you by [bokeru](https://osu.ppy.sh/users/11308433), [chidodou](https://osu.ppy.sh/users/13868581), [Supportive](https://osu.ppy.sh/users/21482068) and [Whithard](https://osu.ppy.sh/users/24552396), this contest revolves around the idea of expression, versatility, and best of all, **expression**! Keep reading to see how you can join in on the fun by participating!
+
+With the first iteration and the first of its kind in osu!, as well as being most inspired from BMS of Fighters (surely something you've heard about before), Kyoku is a contest that will focus on not only beatmapping, but song production as well.
+
+Instead of free song choice, or the usual forced song pick, you'll have to get together with at least one musician (or even be a producer yourself!) in teams of 2–8 (up to 5 mappers and up to 3 musicians) to create a fresh new song AND map!
+
+## Schedule
+
+| Phase | Dates |
+| --: | :-- |
+| Registrations | Now to June 9 |
+| Song production & mapping period | June 10 to August 22 |
+| Judging | August 23 to August 31 |
+| Results | September 1 |
+
+## Song rules
+
+Starting June 10th, 2024, musicians will be allowed to send their song to any of the organisers to check over for approval, under these rules:
```

Comment:
via discord? the organisers would be [bokeru](https://osu.ppy.sh/users/11308433), [chidodou](https://osu.ppy.sh/users/13868581), [Supportive](https://osu.ppy.sh/users/21482068) and [Whithard](https://osu.ppy.sh/users/24552396), right?